cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes how a gamification system will automate the
+process of becoming a `Developer` from a `Newcomer`, as a part of
+this
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5892040252981248).
+
+For the implementation of this process, this project will
+incorporate a gamification system which will allow us to
+track the progress of newcomers and assign some points to
+each of the activities they do. There will be different
+levels based on point thresholds and a newcomer will also
+get some special privileges after the completion of each
+level. There will also be different badges on the basis of
+performed activities by the newcomers.
+
+So, every newcomer at coala will have to complete the processes
+involved in the gamification system in order to become a developer
+at coala or to get the **coala Developer** badge.
+
+## Background
+
+### Newcomer Experience
+
+**The problems we have:**
+
+A good way to keep an Open Source project alive is to get new
+contributors as the old ones will be busy in their own work
+someday and couldn't be able to contribute on the daily basis.
+So, the first and foremost problem we have is, how to attract
+more newcomers to contribute to the community?
+
+But, with increasing amount of newcomers, chaos comes. And it will
+be hard to focus on each and every new contributor. So, we also
+have to find a way to manage the same. Also, to keep the newcomers
+contributing to the community we need to find a way to teach and
+help them so that we can keep working on coala at a fast pace.
+
+We also need to find a way to get more effective feedback from
+the newcomers who have just started using tools we are building.
+And how we can use those feedback to improve coala and its
+processes.
+
+## Why Gamification?
+
+As coala tries to be a welcoming organization for newcomers,
+it gives them clear pathways and as much direct assistance as
+possible. However, it is already known that newcomers face many
+barriers while attempting to contribute to open source for the
+first time, and additional barriers while attempting to
+contribute to projects run by the coala organization.
+
+Some barriers they face include orientation issues that can
+potentially demotivate newcomers from placing their first
+contribution. On the other hand, gamification is widely used
+to engage and motivate people to accomplish tasks and improve
+their performance.
+
+Therefore, the goal of this project is to use gamification to
+orient and motivate newcomers to overcome onboarding barriers
+to contribute and engage with coala and its community.
+
+## Gamification Introduction
+
+There will be four game design elements to help newcomers:
+Activities, Points, Badges and Levels. For each of these
+elements, we will have a set of rules to describe the
+operation of the gamified environment. i.e.:
```

Comment:
remove `i.e.:`

that is grammar and requires to be directly followed by something with a defined end.  but below you have very long sub sections so the reader has no idea where the `i.e.` ends.